src/components/Header.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import styled from "styled-components";
+import LoginButton from "../components/LoginButton";
+
+const Header = styled.div`
+  display: flex;
+  flex-direction: row;
+  width: 375px;
+  height: 64px;
+  color: lightblue;
+  opacity: 44%;
+  border: 1px solid #707070;
+`;
+
+export default Header;
+export function Login() {
```

Comment:
This file exports `Header` and `Login`. Are you sure `Login` is used?
If so, please split it to have only one component exported for ever file to have a cleaner folder structure.